tasks/configure.yml

```diff
@@ -194,3 +194,46 @@
            --execute="update fpr_fprule set enabled={{ item.value.enabled }} where {{ item.value.field_name }}=\"{{ item.key }}\";"
   loop: "{{ lookup('dict',archivematica_src_configure_fprule|default({})) }}"
   when: archivematica_src_configure_fprule is defined
+
+- name: "Configure locations"
+  block:
+    # Get id of the first registered pipeline (id=1)
+    - name: "Configure AM: get default pipeline UUID from SS database"
+      become: "yes"
+      command:  mysql {{ archivematica_src_ss_db_name }} -Ns -e "select uuid from locations_pipeline where id='1';"
+      register: pipeline_uuid
+      tags: "configure-am"
+
+    # Gets the uuid of the first registered space (id=1)
+    - name: "Configure AM: get default Space UUID from SS database"
+      become: "yes"
+      command:  mysql {{ archivematica_src_ss_db_name }} -Ns -e "select uuid from locations_space where id='1';"
+      register: space_uuid
+      tags: "configure-am"
+
+    - name: "Configure AM: get all TS descriptions from SS database"
+      become: "yes"
+      command:  mysql {{ archivematica_src_ss_db_name }} -Ns -e "select description from locations_location;"
+      register: location_descriptions
+      tags: "configure-am"
+
+    - name: "Configure SS: add NFS location: {{ item.location_description }}"
```

Comment:
Hi Santi, the PR looks fine, but I think we need a cosmetic change. This task name should be more generic, something like "Configure SS: add custom locations". The `{{ item.location_description }}` is not changed to its value on every iteration, so we can remote it from name:

```
TASK [external-roles/artefactual-archivematica-src-mysql : Configure SS: add NFS location: {{ item.location_description }}] ************************************************************************
ok: [mamedin-bionic-PR261] => (item={u'location_path': u'/replicator', u'location_purpose': u'RP', u'location_description': u'Replicator location', u'location_default': u'false'})
ok: [mamedin-bionic-PR261] => (item={u'location_path': u'/aipstore', u'location_purpose': u'AS', u'location_description': u'AipStore', u'location_default': u'true'})
ok: [mamedin-bionic-PR261] => (item={u'location_path': u'/transfer-source/', u'location_purpose': u'TS', u'location_description': u'Transfer Source', u'location_default': u'false'})
```